tests/test_authentication.py

```diff
@@ -38,6 +39,14 @@ def test_success_registration(self):
         self.assertIn(
             "You have been registered successfully. Please login", str(res.data))
 
+    def test_user_registration_twice(self):
```

Comment:
It is good practice for a particular test to have one assert

Reply:
What do you mean here? I shouldn't have two methods that deal with registration ?